presto-main/src/main/java/io/prestosql/operator/scalar/timetz/TimeZoneHour.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.operator.scalar.timetz;
+
+import io.prestosql.spi.function.Description;
+import io.prestosql.spi.function.LiteralParameters;
+import io.prestosql.spi.function.ScalarFunction;
+import io.prestosql.spi.function.SqlType;
+import io.prestosql.spi.type.LongTimeWithTimeZone;
+import io.prestosql.spi.type.StandardTypes;
+
+import static io.prestosql.spi.type.DateTimeEncoding.unpackOffsetMinutes;
+import static io.prestosql.type.DateTimes.MINUTES_PER_HOUR;
+
+@Description("Time zone hour of the given time")
```

Comment:
time -> time with time zone ?

Reply:
"time" is not meant to be a type, but a general indication of the kind of temporal value. The type is evident from the function signature as shown in SHOW FUNCTIONS.

Reply:
yeah i understand. 
in fact, i have no problem calling `timestamp with time zone` a "timestamp" for brevity
i have more problem calling `time with time zone` a "time" though, because... it's just a "pair" :)